spec/features/bulkrax_dynamic_import_spec.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "csv"
+
+# rubocop:disable RSpec/InstanceVariable
+RSpec.describe "Bulkrax import", clean: true, slow: true do
+  include CsvWriterHelper
+  include CsvReaderHelper
+
+  let(:user) { create(:user, email: "test@example.com") }
+  let(:depositor) { build_stubbed(:user, email: "batchuser@example.com") }
+
+  let(:number_of_records) { 3 } # Setting this too high will increase the test time for little gain
+
+  after do
+    ["UbiquityTemplateWork", "UvaWork"].each do |model_name|
+      File.delete("spec/fixtures/csv/#{model_name.underscore}_dynamic_data.csv") if File.exist?("spec/fixtures/csv/#{model_name.underscore}_dynamic_data.csv")
```

Comment:
There are a few really long lines of code in here that it would be great if we could get down to <= 120.